lib/errors.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+class UndiciError extends Error {
+  constructor (message) {
+    super(message)
+    this.name = 'UndiciError'
+    this.code = 'UND_ERR'
+  }
+}
+
+class TimeoutError extends UndiciError {
+  constructor (message) {
+    super(message)
+    Error.captureStackTrace(this, TimeoutError)
```

Comment:
I’m sure there is a reason for this. I just don’t know it. 😅

Reply:
https://nodejs.org/api/errors.html#errors_error_capturestacktrace_targetobject_constructoropt :)